### PR TITLE
fix: dont disable delete on user search

### PR DESCRIPTION
**Pull request by magnew**
Closes: #9191 
### Description:

A user reported they couldnt delete users they had searched for. This is a tiny fix. I don't know why the code was like this, but I everything seems to work well now. 

You can test by searching for a user. If there was only one search result, the delete button was being disabled. Now they should be deletable. 

### Reviewer actions

- [ ] I have manually tested the changes in the preview environment
- [ ] I have reviewed the code
- [ ] I understand that "request changes" will block this PR from merging

### CODE REVIEW

**Comment by ZeRego** [comment]
@magnew context: this check was to disable the button when there was just 1 member in the org. Since we introduced search, we might need the check to use the original list with all org members instead of the filtered list.

**Comment by joaoviana** [comment]
On it! 
Will think of a suggestions after @ZeRego's comment

File: packages/frontend/src/components/UserSettings/UsersAndGroupsPanel/UsersView.tsx [hunk 1]
```diff
@@ -398,7 +398,7 @@ const UsersView: FC = () => {
                                     disabled={
                                         user.data?.userUuid ===
                                             orgUser.userUuid ||
-                                        organizationUsers.length <= 1
+                                        organizationUsers.length < 1
```
Comment (joaoviana) [inline packages/frontend/src/components/UserSettings/UsersAndGroupsPanel/UsersView.tsx:401]: Perhaps we can pass a new param to `useOrganizationUsers` to then set the total number of users on `select`, regardless of the search input. So we can

```ts
  disabled={
                                        user.data?.userUuid ===
                                            orgUser.userUuid || totalNumberOfUsers === 1
```

Comment (magnew) [reply packages/frontend/src/components/UserSettings/UsersAndGroupsPanel/UsersView.tsx:401]: @joaoviana and @ZeRego, the admin user can never delete themself from the UI, so there will always be at least one. Isn't that sufficient? If you would like to be more careful here, we can, but I think this case is handled. 

You also can't change the last admin to not be an admin anymore

Screenshot 2024-03-04 at 15 22 01

Comment (joaoviana) [reply packages/frontend/src/components/UserSettings/UsersAndGroupsPanel/UsersView.tsx:401]: Oh I see @magnew

I just tested this: 
- Couldnt' delete myself
- Searched for me and couldn't delete myself
- Searched for others and could delete them
- Deleted everyone except me 
- Couldn't delete myself, and couldn't delete myself while searching

**Review by joaoviana — APPROVED** [review]

**Pull request merged**